src/lib/mermaid/facade.ts

```diff
@@ -24,10 +24,10 @@
   }
 
   private async render(element: Element): Promise<void> {
-    const id = crypto.randomUUID(),
-      code = element.innerHTML,
+    const id = 'mermaid-' + crypto.randomUUID(),
```

Comment:
It's a good practice to prefix IDs with a string to prevent them from starting with a number. However, consider making 'mermaid-' a constant to avoid magic strings in the code. [medium]